iOS User Notification/iOS User Notification/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  iOS User Notification
+//
+//  Created by Jay Buddhdev on 20/07/22.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
```

Comment:
remove extra spaces, there should be only one space at start and end of class of function thought app